.github/workflows/lint-test.yaml

```
@@ -48,5 +49,19 @@ jobs:
         run: kubectl apply -k https://github.com/kubernetes-sigs/gateway-api/config/crd
         if: steps.list-changed.outputs.changed == 'true'
 
+      - name: Apply Prometheus Operator CRDs (unless for charts already installing it)
+        env:
+          CHANGED_LIST: ${{ steps.list-changed.outputs.changed_list }}
+        run: |
+          # Always run unless only changed chart is one of impacted charts
+          echo "List is: $CHANGED_LIST"
+          if [ "$CHANGED_LIST" = '"charts/kube-prometheus-stack"' ] || [ "$CHANGED_LIST" = '"charts/prometheus-operator-crds"' ]; then
+            echo "Skipping install"
+          else
+            helm repo add prometheus-community https://prometheus-community.github.io/helm-charts --force-update
+            helm install prometheus-operator-crds prometheus-community/prometheus-operator-crds
```

```suggestion
            helm repo add prometheus-community https://prometheus-community.github.io/helm-charts --force-update
            helm install prometheus-operator-crds prometheus-community/prometheus-operator-crds
```

Please use the oci chart here to avoid the slow helm repo add command.

---

Interesting. Should we update README.md files then (in another PR)?

---

Thanks. Updated https://github.com/desaintmartin/helm-charts-3/pull/3 to test the new changes.